### PR TITLE
Enhance start server

### PR DESCRIPTION
**Pull request by FlavianEng**

### CODE REVIEW

**Comment by FlavianEng** [comment]
Task linked: [CU-3aybt6n Enhance start server](https://app.clickup.com/t/3aybt6n)

File: src/domains/character/character.controller.ts [hunk 1]
```diff
@@ -24,12 +24,12 @@ export default class CharacterController {
...
-      throw new ApiError({
-        code: ErrorCodes.WRONG_PARAMS,
-        detail: (error as { message: string }).message,
-        instance: this.constructor.name,
-        title: 'Fail to insert character',
-      });
+      throw new BadRequestException(
+        new ApiError({
+          code: ErrorCodes.INSERTION_FAILED,
+          message: 'Fail to insert boat.',
```
Comment (MaximeWrlz) [inline src/domains/character/character.controller.ts:30]: Replace "boat" by "character"

File: src/domains/game/game.controller.ts [hunk 2]
```diff
@@ -22,12 +22,12 @@ export default class GameController {
...
-      throw new ApiError({
-        code: ErrorCodes.WRONG_PARAMS,
-        detail: (error as { message: string }).message,
-        instance: this.constructor.name,
-        title: 'Fail to insert game',
-      });
+      throw new BadRequestException(
+        new ApiError({
+          code: ErrorCodes.INSERTION_FAILED,
+          message: 'Fail to insert boat.',
```
Comment (MaximeWrlz) [inline src/domains/game/game.controller.ts:28]: Replace "boat" by "game"

File: src/domains/level/level.controller.ts [hunk 3]
```diff
@@ -22,12 +22,12 @@ export default class LevelController {
...
-      throw new ApiError({
-        code: ErrorCodes.WRONG_PARAMS,
-        detail: (error as { message: string }).message,
-        instance: this.constructor.name,
-        title: 'Fail to insert level',
-      });
+      throw new BadRequestException(
+        new ApiError({
+          code: ErrorCodes.INSERTION_FAILED,
+          message: 'Fail to insert boat.',
```
Comment (MaximeWrlz) [inline src/domains/level/level.controller.ts:28]: Replace "boat" by "level"

File: src/domains/media-with-theme/media-with-theme.controller.ts [hunk 4]
```diff
@@ -26,12 +26,12 @@ export default class MediaWithThemeController {
...
-      throw new ApiError({
-        code: ErrorCodes.WRONG_PARAMS,
-        detail: (error as { message: string }).message,
-        instance: this.constructor.name,
-        title: 'Fail to insert user',
-      });
+      throw new BadRequestException(
+        new ApiError({
+          code: ErrorCodes.INSERTION_FAILED,
+          message: 'Fail to insert boat.',
```
Comment (MaximeWrlz) [inline src/domains/media-with-theme/media-with-theme.controller.ts:32]: Replace "boat" by "media with theme"

File: src/domains/media/media.controller.ts [hunk 5]
```diff
@@ -22,12 +22,12 @@ export default class MediaController {
...
-      throw new ApiError({
-        code: ErrorCodes.WRONG_PARAMS,
-        detail: (error as { message: string }).message,
-        instance: this.constructor.name,
-        title: 'Fail to insert media',
-      });
+      throw new BadRequestException(
+        new ApiError({
+          code: ErrorCodes.INSERTION_FAILED,
+          message: 'Fail to insert boat.',
```
Comment (MaximeWrlz) [inline src/domains/media/media.controller.ts:28]: Replace "boat" by "media"

File: src/domains/theme/theme.controller.ts [hunk 6]
```diff
@@ -22,12 +22,12 @@ export default class ThemeController {
...
-      throw new ApiError({
-        code: ErrorCodes.WRONG_PARAMS,
-        detail: (error as { message: string }).message,
-        instance: this.constructor.name,
-        title: 'Fail to insert theme',
-      });
+      throw new BadRequestException(
+        new ApiError({
+          code: ErrorCodes.INSERTION_FAILED,
+          message: 'Fail to insert boat.',
```
Comment (MaximeWrlz) [inline src/domains/theme/theme.controller.ts:28]: Replace "boat" by "theme"

File: src/domains/user/user.controller.ts [hunk 7]
```diff
@@ -22,12 +22,12 @@ export default class UserController {
...
-      throw new ApiError({
-        code: ErrorCodes.WRONG_PARAMS,
-        detail: (error as { message: string }).message,
-        instance: this.constructor.name,
-        title: 'Fail to insert user',
-      });
+      throw new BadRequestException(
+        new ApiError({
+          code: ErrorCodes.INSERTION_FAILED,
+          message: 'Fail to insert boat.',
```
Comment (MaximeWrlz) [inline src/domains/user/user.controller.ts:28]: Replace "boat" by "user"

File: src/domains/weapon/weapon.controller.ts [hunk 8]
```diff
@@ -22,12 +22,12 @@ export default class WeaponController {
...
-      throw new ApiError({
-        code: ErrorCodes.WRONG_PARAMS,
-        detail: (error as { message: string }).message,
-        instance: this.constructor.name,
-        title: 'Fail to insert weapon',
-      });
+      throw new BadRequestException(
+        new ApiError({
+          code: ErrorCodes.INSERTION_FAILED,
+          message: 'Fail to insert boat.',
```
Comment (MaximeWrlz) [inline src/domains/weapon/weapon.controller.ts:28]: Replace "boat" by "weapon"

**Review by MaximeWrlz — CHANGES_REQUESTED** [review]

**Review by MaximeWrlz — DISMISSED** [review]

**Review by MaximeWrlz — APPROVED** [review]

**Review by MASBERNATQ — APPROVED** [review]

**Pull request merged**